.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release
+
+on:
+  pull_request:
+    types: [closed]
+    paths:
+      - '.github/project.yml'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: release
+    if: ${{github.event.pull_request.merged == true}}
+
+    steps:
+      - uses: radcortez/project-metadata-action@master
+        name: Retrieve project metadata
+        id: metadata
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          metadata-file-path: '.github/project.yml'
+
+      - uses: actions/checkout@v2
+
+      - name: Import GPG key
```

Comment:
What does it do exactly? 
Are we sure we need this?

Reply:
This is needed to decrypt the Maven settings file where the OSS Jira credentials are stored. See https://help.sonatype.com/repomanager2/staging-releases/configuring-your-project-for-deployment